.github/workflows/maven.yml

```diff
@@ -6,6 +6,7 @@ on:
       - main
   pull_request:
     types: [ opened, synchronize, reopened ]
+  merge_group:
```

Comment:
It is according to github documentation? Outdated knowledge from the LLM? 😊

Reply:
> It is according to github documentation? Outdated knowledge from the LLM? 😊

That could be the case. Or does _merge_group:_ need _types:_ as well? Referring to [https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#merge_group](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#merge_group) 

So this might solve it? 🤔

```
pull_request:
    types: [opened, synchronize, reopened]
merge_group:
    types: [checks_requested]
```

Reply:
> > It is according to github documentation? Outdated knowledge from the LLM? 😊
> 
> That could be the case. Or does _merge_group:_ need _types:_ as well? Referring to https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#merge_group
> 
> So this might solve it? 🤔
> 
> ```
> pull_request:
>     types: [opened, synchronize, reopened]
> merge_group:
>     types: [checks_requested]
> ```

It is possible that the ellipsis dev bot just isn't updated to recognize the _'merge_group'_ event, especially if it's a relatively new feature. What I could find on a GitHub community discussions, was about _'merge_group'_ posted a year ago [“Pull Request Merge Queue Public Beta Feedback”](https://github.com/orgs/community/discussions/46757). So this could explain why the bot doesn't recognize it? 

I’ve also been peeping around in different projects, some have _'merge_group'_ left empty, I guess to trigger for all types of merge events? While others used filters like _'branches'_.
Here is a screenshot from two different projects using _'merge_group'_:

<img width="987" alt="Screenshot 2024-02-15 at 18 38 38" src="https://github.com/fungover/thunder/assets/143100902/872e8fb0-5cdf-427d-b538-add95ceaab36">

Reply:
As I understand it the only type that merge group supports at the moment is checks_requested.
So we don't need to put it there but it can be good to do it because when new types are added in the future we know that they won't start triggering our job.

Reply:
Added type as in the documentation.